### PR TITLE
OP-948 fix count of medical stock when moving to another ward

### PR DESCRIPTION
**Pull request by dbmalkovsky**
This is the hotfix version

See OP-948

### CODE REVIEW

File: src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java [hunk 1]
```diff
@@ -246,6 +246,7 @@ protected boolean updateStockWardQuantity(
 				medicalWard.setLot(movement.getLot());
 				repository.save(medicalWard);
             }
+	        repository.updateOutQuantity(Math.abs(qty), ward, medical, lot);
```
Comment (mwithi) [inline src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java:249]: maybe indentation?

Comment (dbmalkovsky) [reply src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java]: Fixed the indentation and while I was at it corrected some formatting in that area too.

Comment (mwithi) [reply src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java]: saw it, thanks!

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**